lib/double_bag_ftps.rb

```diff
@@ -109,6 +109,10 @@ def transfercmd(cmd, rest_offset = nil)
       conn = ssl_socket(conn) # SSL connection now possible after cmd sent
     else
       sock = makeport
+      # Before ruby-2.3.0, makeport did a sendport automatically
+      if RUBY_VERSION >= "2.3"
```

Comment:
This version check needs to be appropriate for affected versions.  Apparently 2.2.* series is also affected.